app/views/followups/_followup.html.erb

```diff
@@ -78,20 +78,43 @@
     <% status = @info_request.calculate_status %>
     <% if status == 'waiting_response_overdue' %>
       <p>
-        <%= _('The response to your request has been <strong>delayed</strong>. ' \
-              'You can say that, by law, the authority should normally have ' \
-              'responded <strong>promptly</strong> and by ' \
-              '<strong>{{date}}</strong>',
-              :date=>simple_date(@info_request.date_response_required_by)) %>
+        <% if @info_request.public_body.not_subject_to_law? %>
+          <%= _('The response to your request has been <strong>delayed' \
+                '</strong>. Althouth the authority has no legal obligation ' \
+                'to reply, they should normally have responded by ' \
+                '<strong>{{date}}</strong>',
+                :date=>simple_date(@info_request.date_response_required_by)) %>
 
-        (<%= link_to _('details'), "#{help_requesting_path}#quickly_response" %>).
```

Comment:
It would have been nice to make a commit that clean this up before propagating it further…

```diff
# Fix incorrect use of linking to anchor
- (<%= link_to _('details'), "#{help_requesting_path}#quickly_response" %>).
+ (<%= link_to _('details'), help_requesting_path(anchor: 'quickly_response') %>).
```